docs/docs/guides/exporting-logs.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Exporting logs to external service
+sidebar_label: Exporting logs to external service
+slug: /exporting-logs
+sidebar_position: 15
+---
+
+This guide assumes that you have [Kurtosis installed](../get-started/installing-the-cli.md).
+
+Kurtosis comes with a logs aggregator component that aggregates logs from services across enclaves. This component uses
+Vector under the hood. Logs aggregator can be configured independently for each cluster, in the Kurtosis [config file](../cli-reference/index.md#configuration-file-path).
+
+Kurtosis uses the notion of "sinks" to describe the location where you want your logs exported. Sink configurations are
+forwarded as-is (with some exceptions, see below) to Vector, therefore Kurtosis can export to all sinks that Vector supports. 
```

Comment:
maybe link to Vector here and mention that Kurtosis allows configuring vector sinks?